model/lifecycle.go

```diff
@@ -1596,6 +1597,7 @@ func addNewBuilds(ctx context.Context, creationInfo TaskCreationInfo, existingBu
 		for _, t := range tasks {
 			if t.Activated {
 				newActivatedTaskIds = append(newActivatedTaskIds, t.Id)
+				numEstimatedActivatedGeneratedTasks += utility.FromIntPtr(t.EstimatedNumActivatedGeneratedTasks)
```

Comment:
should we call `setGenerateTaskEstimate` if t.estimated is 0 and it should generate tasks?

Reply:
Yes, actually just left [this comment](https://github.com/evergreen-ci/evergreen/pull/8060#discussion_r1668795481) on your other open PR